src/newrelic/app_test.go

```diff
@@ -375,7 +375,7 @@ func TestConnectPayloadEncoded(t *testing.T) {
 		`"metadata":{"NEW_RELIC_METADATA_ONE":"one","NEW_RELIC_METADATA_TWO":"two"},` +
 		`"identifier":"one;two",` +
 		`"utilization":{"metadata_version":1,"logical_processors":22,"total_ram_mib":1000,"hostname":"some_host"},` +
-		`"event_harvest_config":{"report_period_ms":60000,"harvest_limits":{"error_event_data":100,"analytic_event_data":10000,"custom_event_data":10000,"span_event_data":2323}}` +
+		`"event_harvest_config":{"report_period_ms":60000,"harvest_limits":{"error_event_data":100,"analytic_event_data":10000,"custom_event_data":10000,"span_event_data":2323,"log_event_data":10000}}` +
```

Comment:
I have this in my current PR - I think we need to set the `info.AgentEventLimits.SpanEventConfig.Limit` to a specific value and check for that value in the generated JSON.
Perhaps we need another test to verify the defaults if agent limits are not supplied?

Reply:
SpanEvent or LogEvent?